src/Symfony/Component/Security/Core/Authorization/Voter/VoteInterface.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Security\Core\Authorization\Voter;
+
+/**
+ * A VoteInterface Object contain information about vote, access/score, messages.
+ *
+ * @author Roman JOLY <eltharin18@outlook.fr>
+ */
+interface VoteInterface
+{
+    public function __construct(int $access, string|array $messages = [], array $context = []);
```

Comment:
It feels weird to have the ctor here, as an interface is about consuming not creating an object